src/test/java/reposense/parser/ArgsParserTest.java

```diff
@@ -95,12 +99,6 @@ public void parse_viewOnly_success() throws ParseException, IOException {
         String input = String.format("-view %s", OUTPUT_DIRECTORY_ABSOLUTE);
         CliArguments cliArguments = ArgsParser.parse(translateCommandline(input));
         Assert.assertTrue(Files.isSameFile(OUTPUT_DIRECTORY_ABSOLUTE, cliArguments.getReportDirectoryPath()));
-        // Optional arguments have default values
-        Assert.assertEquals(Optional.empty(), cliArguments.getSinceDate());
-        Assert.assertEquals(Optional.empty(), cliArguments.getUntilDate());
-        Assert.assertEquals(ArgsParser.DEFAULT_REPORT_NAME, cliArguments.getOutputFilePath().getFileName().toString());
-        Assert.assertEquals(ArgsParser.DEFAULT_FORMATS, cliArguments.getFormats());
-        Assert.assertNull(cliArguments.getConfigFilePath());
```

Comment:
In case of `view` only, I skipped the actual object construction, so all items will not have default values but instead they will be null. 

Since we are not using the values, can we leave the test out?